lib/java-extras/src/main/java/org/triplea/java/collections/CollectionUtils.java

```diff
@@ -161,6 +162,15 @@ public static <T> T getAny(final Iterable<T> elements) {
     return elements.iterator().next();
   }
 
+  /** Like getAny, but skip elements if possible */
+  public static <T> T getAt(final Iterable<T> elements, final Integer count) {
+    Iterator<T> iterator = elements.iterator();
```

Comment:
I don't think there's any guarantee that iterator() will return elements in the same order each time. It can, if the collection is a List, for example, but not if it's a Set (which in this case it is).

So for example, you can have a scenario where the container has elements A, B, C and the first time you call iterator(), it will give you A, B, C order, but the next time, order will be B, A, C. So any calling logic that expects getAt() to be consistent won't work correctly.

If you really want these semantics, you should just convert the collection to a List first outside and just use standard .get().